core/controller/Dockerfile

```diff
@@ -3,14 +3,17 @@
 
 FROM scala
 
+ENV SWAGER_DOWNLOAD_SHA256 3d7ef5ddc59e10f132fe99771498f0f1ba7a2cbfb9585f9863d4191a574c96e7
+ENV SWAGER_VERSION 3.6.0
```

Comment:
typo: Swagger

Reply:
Oups sorry! Thanks for seeing this. Just pushed again